src/profile-logic/import/instruments/MaybeCompressedReader.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import * as pako from 'pako';
```

Comment:
```
// Top level file is just a mixin of submodules & constants
'use strict';

var assign    = require('./lib/utils/common').assign;

var deflate   = require('./lib/deflate');
var inflate   = require('./lib/inflate');
var constants = require('./lib/zlib/constants');

var pako = {};

assign(pako, deflate, inflate, constants);

module.exports = pako;
```

This is the structure of `node_modules/pako/index.js` file. There is no named export.
That's why I did this.

Maybe I can replace that line with this: `import pako from 'pako';`.

Reply:
Does `import inflate from 'pako/lib/inflate';` work?

Reply:
@mstange Thanks for the suggestion.
But it didn't work.

Error:
<img width="863" alt="Screenshot 2019-08-25 at 11 23 00 PM" src="https://user-images.githubusercontent.com/16744065/63653815-6af1e800-c78f-11e9-9603-77d28a292ecc.png">

Reply:
But doesn't `import { inflate } from 'pako'`? Normally webpack knows how to deal with old-style libraries too.

Reply:
Woot! It worked!
But I have to keep the content of `pako_vx.x.x.js` like this:

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
// @flow

declare module 'pako' {
  declare module.exports: any;
}
```

Pushing the above change.
cc: @canaltinova

Reply:
I think we can do better, but because your summer of code is finished I don't want to hold you more.
@canaltinova do you think you can help defining a better libdef?
Otherwise we can file an issue so that somebody can look at it later.

Reply:
@julienw, yeah GSoC is finished, but I am available to contribute as before!

Maybe I will take a 1-week break, then I will continue my contribution :)